i18n/fr/docusaurus-plugin-content-docs/version-23.04/reporting/installation.md

```diff
@@ -339,7 +339,7 @@ apt update
 Puis installez Centreon MBI:
 
 ```shell
-apt update && apt install centreon-bi-server
+apt install centreon-bi-server
```

Comment:
not the same update command as for version 22.10
it looks like EN and FR are not the same
this needs to be cleaned up